config/job_conf.xml.sample_advanced

```diff
@@ -703,6 +722,17 @@
                  Disable venv by setting the value to false.
             -->
         </destination>
+        <destination id="chronos_dest" runner="chronos">
+            <param id="docker_enabled">true</param>
+            <param id="docker_memory">512</param>
+            <param id="docker_cpu">2</param>
+            <param id="volumes">/directory/</param>
+            <!-- Directory which is mounted to the container and is parent of
+                 the  `job_working_directory`, `file_path`, `new_file_path`
+                 directories -->
```

Comment:
Can you update this comment to mention reference data as well - everything mentioned in ``.loc`` files (if they are used by these jobs) need to be included as well.